__tests__/app.test.js

```diff
@@ -233,7 +233,7 @@ describe("Patch /api/reviews/:review_id", () => {
   });
 });
 
-describe("get /api/reviews", () => {
+describe("get /api/reviews?category=blah", () => {
```

Comment:
I would maybe get rid of the blah, as it suggests you're always looking for `blah`.
it might make sense to you, but could be confusing to others looking at your code.